packages/itwin/ec3-widget/src/components/templateClient.tsx

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
+  getTemplatesT(): Template[] {
+
+    var templates: Template[] = [];
+
+    for (var i = 0; i < localStorage.length; i++) {
+      const key = localStorage.key(i);
+      if (key && key.startsWith("sel.")) {
+        const value = localStorage.getItem(key);
+
+        if (value) {
+          const sel = JSON.parse(value);
+          templates.push(sel);
+        }
+      }
+    }
+
+    return templates;
+
+    //templates[0] =
+    //const text = localStorage.getItem("template");
+
+    //return JSON.parse(text ?? "");
+  }
+
+
+  getTemplateT(templateId: string): Template | undefined {
+
```

Comment:
No trailing or leading empty lines in methods and no double empty lines.

Reply:
Unnecessary empty lines have been removed